### PR TITLE
Update README.md

### LINKED ISSUE #965: Update in-repo docs for 2.0.0
2015 support was dropped from the package, but not all of the docs were updated.

### PR DESCRIPTION
**Pull request by ILMTitan**
Fixes #965

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,70 @@
...
+[![Build status][BuildBadge]][Build]
+[![Code Coverage][CoverageBadge]][Coverage]
 
 # Visual Studio Extension for Google Cloud Platform
-This repo contains the source code for the Visual Studio Extension for Google
-Cloud Platform.
 
-To build and install the extension you will need to have Visual Studio 2015, with Update 3 as the recommended update level. You will need the the Visual Studio SDK feature installed to be able to develop and build the extension. After that just open the .sln file and build it, that is really it. If you get an error during the restoration of the NuGet packages ensure that you have enabled nuget.org as a
-source in the NuGet Package Manager.
+This repo contains the source code for Cloud Tools for  Visual Studio, an extension
```
Comment (iantalarico) [inline README.md]: nit: double space after for

File: README.md [hunk 2]
```diff
@@ -1,49 +1,70 @@
...
+for integrating Google Cloud Platform into Visual Studio.
```
Comment (iantalarico) [inline README.md]: is it "the Google Cloud Platform"?

File: README.md [hunk 3]
```diff
@@ -1,49 +1,70 @@
...
-The Visual Studio extension depends on the Cloud SDK for certain functionality, make sure that you have at least version 164.0.0 of the Cloud SDK installed when running the extension.
 
-You can install the Google Cloud SDK from <https://cloud.google.com/sdk/>. You will also need to make sure that you have the "beta" and "kubectl" components.
-* The "beta" component is used for the deployment ASP.NET Core apps to App Engine Flex.
-* The "kubectl" component is used for interactions with Google Container Engine.
+The Visual Studio extension depends on the Cloud SDK for certain functionality, make
+sure that you have at least version 174.0.0 of the Cloud SDK installed when running the extension.
+
+You can install the Google Cloud SDK from <https://cloud.google.com/sdk/>. You will
+also need to make sure that you have **kubectl** component. It is used for interactions with Google Container Engine.
```
Comment (iantalarico) [inline README.md]: nit: the kubectl

Comment (iantalarico) [inline README.md]: Google Kubernetes Engine (this was renamed)

File: README.md [hunk 4]
```diff
@@ -1,49 +1,70 @@
...
 To get help on using the Visual Studio Extension, please log an issue with this
 project. While we will eventually be able to offer support on Stack Overflow or
 a Google+ community, for now your best bet is to report issues in this Github
 project.
 
+## Build
+
+To build the extension you will need to have Visual Studio 2017. You will need the
+the Visual Studio extension development toolset installed to be able to develop and build the extension.
+After that just open the .sln file and build it it. If you get an error during the
```
Comment (quoctruong) [inline README.md]: Extra `it` after `build it`.

File: README.md [hunk 5]
```diff
@@ -1,49 +1,70 @@
...
+restoration of the NuGet packages ensure that you have enabled nuget.org as a source in the NuGet Package Manager.
```
Comment (quoctruong) [inline README.md]: Should be `NuGet packages, ensure that`.

File: README.md (hunk 4 above)
Comment (iantalarico) [inline README.md]: maybe: Then open and build the main soultion file: GoogleCloudExtension.sln (can't remember the exact name)

File: README.md (hunk 2 above)
Comment (ILMTitan) [reply README.md]: "Google Cloud Platform", used as a brand name, would not need "the". You could read it as the "Cloud Platform" that is "Google", but that makes things less clear, and is not how it is used in the official docs:

> Build, innovate, and scale with Google Cloud Platform

File: README.md (hunk 1 above)
Comment (ILMTitan) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (ILMTitan) [reply README.md]: Done.

Comment (ILMTitan) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (ILMTitan) [reply README.md]: Done.

Comment (ILMTitan) [reply README.md]: Done.

File: README.md (hunk 5 above)
Comment (ILMTitan) [reply README.md]: Done.

**Comment by ILMTitan** [comment]
PTAL

**Pull request merged**